spacy_llm/backends/integration/hf/base.py

```diff
@@ -16,16 +18,19 @@ class HuggingFaceBackend(abc.ABC):
     def __init__(
         self,
         model: str,
-        config: Dict[Any, Any],
+        config: Dict[str, Dict[Any, Any]],
```

Comment:
Could we add `init` and `run` configs as separate parameters here instead?

Reply:
Resolving this, let's continue in https://github.com/explosion/spacy-llm/pull/151#discussion_r1213009097.